### PR TITLE
Add PPP

### LINKED ISSUE #230: Write PPP for Long

### PR DESCRIPTION
**Pull request by longngng**

### CODE REVIEW

File: docs/team/longngng.md [hunk 1]
```diff
@@ -2,5 +2,48 @@
...
+* Added details and sequence diagrams for `list` command in `Notebook` mode
+* Added details and sequence diagrams for `list` command in `Timetable` mode
+* Added details and sequence diagrams for `ModeSwitch` feature
+* Added details for `Task` components and `Notebook` components
+
+### Contributions to team-based tasks
+
+* I created the first release of the jar file.
+* I refactored the code once at the beginning and introduce the AppState object to improve the use of OOP principles.
+* I created a demo video for the first release to present during the tutorial
```
Comment (franceneee) [inline docs/team/longngng.md]: nice haha

File: docs/team/longngng.md [hunk 2]
```diff
@@ -2,5 +2,48 @@
...
+### Code contributed
+
+See the [RepoSense Dashboard here](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=longngng&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other) for my contributions to Zer0Note. 
+
+### Enhancements implemented
+
+* Implemented the `list` command for timetable mode in v1.0 
+* Implemented `mode` command in v1.0
+* Implemented the `list` command for notebook mode in v2.0 
+* Implemented `list /urgent` command in v2.0 to allow users view their top three upcoming deadlines.
```
Comment (franceneee) [inline docs/team/longngng.md]: Maybe you could add more details? like the example PPP, you can justify why there's a need for these ?  https://se-education.org/addressbook-level3/team/johndoe.html

File: docs/team/longngng.md (hunk 1 above)
Comment (yAOwzers) [reply docs/team/longngng.md]: LOL

Comment (yAOwzers) [reply docs/team/longngng.md]: Maybe you can link like the Pull requests?

**Review by yAOwzers — APPROVED** [review]

File: docs/team/longngng.md (hunk 2 above)
Comment (longngng) [reply docs/team/longngng.md]: Yep, I was looking for this 🙇

Comment (longngng) [reply docs/team/longngng.md]: Will update.

**Review by franceneee — APPROVED** [review]
Haha nice, it has more details now. :)

**Pull request merged**